cloud/pod_util.go

```diff
@@ -95,6 +100,46 @@ func ExportPodResources(info pod.ResourceInfo) cocoa.ECSPodResources {
 	return *res
 }
 
+// ExportPodContainerDef exports the ECS pod container definition into the equivalent cocoa.ECSContainerDefintion.
+func ExportPodContainerDef(opts pod.TaskContainerCreationOptions) (*cocoa.ECSContainerDefinition, error) {
+	if opts.Image == "" {
```

Comment:
You don't need to check this. It's already validated by the REST route and in cocoa.

Reply:
Cool, since some of the tests in `pod_util_test` deal with input validation that is in cocoa, I took those out along with taking this piece of extra validation out.